src/main/java/de/retest/web/DefaultValuesProvider.java

```diff
@@ -74,12 +74,12 @@ public Serializable getDefaultValue( final IdentifyingAttributes comp, final Str
 
 	public boolean isDefault( final String tag, final String attribute, final String attributeValue ) {
 		final String defaultValue = getDefaultValue( tag, attribute );
-		if ( defaultValue != null ) {
-			return defaultValue.equalsIgnoreCase( attributeValue );
-		}
 		if ( attributeValue == null || attributeValue.trim().isEmpty() ) {
 			return true;
 		}
+		if ( attributeValue.equalsIgnoreCase( defaultValue ) ) {
```

Comment:
Damn, I overlooked that. We want to be able to say, that null is not the default value...